moveit_planners/pilz_industrial_motion_planner/src/pilz_industrial_motion_planner.cpp

```diff
@@ -123,19 +123,24 @@ CommandPlanner::getPlanningContext(const planning_scene::PlanningSceneConstPtr&
                                    moveit_msgs::MoveItErrorCodes& error_code) const
 {
   ROS_DEBUG_STREAM("Loading PlanningContext for request\n<request>\n" << req << "\n</request>");
-
-  // Check that a loaded for this request exists
-  if (!canServiceRequest(req))
+  std::string planner_id = req.planner_id;
+  if (planner_id.empty() && context_loader_map_.find("PTP") != context_loader_map_.end())
+  {
+    planner_id = "PTP";
+    ROS_INFO_STREAM("Using default planner: " + planner_id);
```

Comment:
```suggestion
    ROS_WARN_STREAM("Empty planner_id in the request. Using default planner: " + planner_id);
```

Reply:
I am undecided to merge this PR. I can follow both arguments. Consider it as a proposal. 
Other maintainers might comment.
Alternatively / additionally, we should configure a default planner.